libs/libshout/patches/150-openssl-1.1.patch

```diff
@@ -9,7 +11,12 @@
  	SSL_load_error_strings();
  	SSLeay_add_all_algorithms();
   	SSLeay_add_ssl_algorithms();
```

Comment:
Add `#define TLS_client_method TLSv1_client_method` here, otherwise, it will not work with openssl 1.0.  You could use the more general `SSLv23_client_method`, which I would prefer, but we would be changing functionality here.  
Here's the warning that exposes the bug:
```
tls.c: In function 'tls_setup':
tls.c:75:9: warning: implicit declaration of function 'TLS_client_method'; did you mean 'DTLS_client_method'? [-Wimplicit-function-declaration]
  meth = TLS_client_method();
         ^~~~~~~~~~~~~~~~~
         DTLS_client_method
tls.c:75:7: warning: assignment makes pointer from integer without a cast [-Wint-conversion]
  meth = TLS_client_method()

```
I completely missed this one when I made the original patch.  Thanks for reviewing it.

Reply:
Patch is simpler this way. Please verify.

Reply:
I'm certain.  There's no mention of the methods in docs, nor the header files.  I've even run `egrep '[^D]TLS_(server|client)_method'` on the openssl-1.0.2g sources to se if it's hidden somewhere, but it turns out nothing--notice the [^D] as there are DTLS_*method.   I haven't run test it, but I imagine we'll get missing symbols when linking against the lib.